users-service/src/main/java/org/springboardfoundation/usersservice/service/impl/UserServiceImpl.java

```diff
@@ -18,6 +20,11 @@ public class UserServiceImpl implements UserService {
 	public UserDto saveUser(UserDto userDto) {
 		// copy all data into new user object pass that to save
 		User user = UserMapper.MAPPER.map(userDto);
+
+		UUID uuid=UUID.randomUUID();
+
+		user.setUserIdentifier((uuid.toString()));
```

Comment:
From the value got from `uuid.toString()` please remove the `-` from the string before setting as the identifier.